.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+[Short description explaining the high-level reason for the pull request]
+
+## Additions
+
+-
+
+## Removals
+
+-
+
+## Changes
+
+-
+
+## Testing
+
+1.
+
+## Screenshots
+
+
+## Notes
+
+-
+
+## Todos
+
+-
+
+## Checklist
+
+- [ ] PR has an informative and human-readable title
+- [ ] Changes are limited to a single goal (no scope creep)
+- [ ] Code can be automatically merged (no conflicts)
+- [ ] Code follows project standards (link if applicable)
+- [ ] Passes all existing automated tests
+- [ ] Any _change_ in functionality is tested
+- [ ] New functions are documented (with a description, list of inputs, and expected output)
+- [ ] Placeholder code is flagged / future todos are captured in comments
+- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
+- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
+- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:
+
+## Testing checklist
+
+### Target Environment support
+
+- [ ] Windows
+- [ ] Linux
+- [ ] Browser
+
+### Accessibility
+
+- [ ] Keyboard friendly
+- [ ] Screen reader friendly
+
+### Other
```

Comment:
This section might need to be trimmed / made more generic. Otherwise this file is good.